docs/source/ddp_comm_hooks.rst

```diff
@@ -81,7 +81,9 @@ PowerSGD Hooks
     compressed communication and improve accuracy.
 
 .. warning ::
-    The current implementation may cause gradient overflow for FP16 input.
```

Comment:
Is this no longer the case?

Reply:
Yes, previously I thought the error was due to overflow, but in that case the value should be infinity instead of NaN.